app/javascript/src/api/account.js

```diff
@@ -5,6 +5,78 @@
 import endPoints from './endPoints';
 
 export default {
+  /**
+   * handleGet
+   * @param dataType {string} [dataType='agent', 'label', 'inbox'] type of data requested
+   * @returns {promise}
+   */
+  handleGet(dataType) {
+    let urlData;
+    switch (dataType.toLowerCase()) {
```

Comment:
@ryanomor We could move the switch part 
```
const computeGetEnpoint = ( dataType) => {
let urlData;
  switch (dataType.toLowerCase()) {
    .......
}
return urlData;
```
as a separate function and return the value `urlData` at the end of the fucntion to get rid of the consistent return error. Also avoid the return in `default` section of switch case.
read more here https://eslint.org/docs/rules/consistent-return

Reply:
We could move the `switch` statements out into a function, but I'm wondering if all we need to do is take out the `return` in the `default` case?